### DISCUSSION TITLE
Share -> HTML File

### DISCUSSION THREAD

**Discussion by xavierve** [gsantner/markor#2000] [Ideas] [answered]
Hi There,
By now we can share as
- Plaint Text
- File (file.md)
- PFD
- HTML Code
- HTML Code (plain text)
- Screenshot

ok, that's great, but why not exporting like "HTML File"?
Easier to open, easier to read, easier to use.

**Comment by gsantner** [comment, marked as answer]
> ok, that's great, but why not exporting like "HTML File"?

Probably because nobody needed that so far or showed any interest into it.
But I'd say there are plenty of options and it's rather bloat to have 
three different options to get the same HTML text out.

If you want changes here, please feel free to make a merge request. But don't add a third option, rather rework one other.

**Reply by alexandre-mbm** [reply]
How to do it as tool?
